src/views/about.hbs

```diff
@@ -40,11 +44,13 @@
       </div>
 
       <li class="f5 black-60 db mb2 tracked pt3 pb3 mr2"><span class='li mr2 w2'>&rarr;</span><label for="about__uncomfortable">What makes you feel uncomfortable or anxious?</label>
+        {{> hide_button}}
         <br>
         <textarea class="mt3 w-100 mw7 vh-5 ba b--black-30 br3 h4" type="text" name="uncomfortable" rows="8" cols="80" placeholder="Write here please" id="about__uncomfortable">{{data.uncomfortable}}</textarea>
       </li>
 
       <li class="f5 black-60 db mb2 tracked pt3 pb3 mr2"><span class='li mr2 w2'>&rarr;</span><label for="about__safe">What helps you feel safe and calm?</label>
+        {{> hide_button}}
         <br>
         <textarea class="mt3 w-100 mw7 vh-5 ba b--black-30 br3 h4" type="text" name="safe" rows="8" cols="80" placeholder="Write here please" id="about__safe">{{data.safe}}</textarea>
       </li>
```

Comment:
great work on the hide_button, but does it feel a little UNDRY to you? I am wondering if you have thought about have a partial for this, running a for loop and then passing information to it? As the list items are very similar. 

@polyccon thoughts?

Reply:
The reason is that is that previously saved information gets loaded into each textarea and the data is different for each question, if that's what you're referring to? Also not all pages have the exact same html structure so it would be hard to generalise a larger portion of that into a partial, but we can revisit this point in the future at some point.